root/release/edit/information.tt

```diff
@@ -310,6 +310,7 @@
     </p>
     <!-- ko with: target() && target().barcode -->
     <p data-bind="visible: message, text: message"></p>
+    <p data-bind="visible: existing, html: existing"></p>
     <p class="field-error" data-bind="visible: error() || confirmed()">
       <label>
         <strong>
```

Comment:
Not directly related to the PR, but I think it would make more sense to have the 'If you do not know whether' block a few lines lower from here to be moved line 310, before the barcode validation messages

Reply:
I'd suggest moving it in a separate commit (in the same PR) if so :)

Reply:
Done